app/src/main/java/com/hover/stax/inapp_banner/BannerUtils.kt

```diff
@@ -14,7 +15,7 @@ import kotlin.properties.Delegates
 
 class BannerUtils(val context: Context) : KoinComponent {
 
-    private val repo: DatabaseRepo by inject()
+    private val repo: AccountRepo by inject()
```

Comment:
Is this safe?

Reply:
Yes, it is; as long as it's been declared in the module.